src/main/java/project/TenderParserServiceApplication.java

```diff
@@ -10,4 +10,5 @@ public static void main(String[] args) {
 		SpringApplication.run(TenderParserServiceApplication.class, args);
 	}
 
+	// empty comment for test commit - delete it!
```

Comment:
I will approve it. But please delete it in the next PR

Reply:
Test Comment)